packs/utils/path.ts

```diff
@@ -1,35 +1,90 @@
 import { Account } from "$store/packs/accounts/configStore.ts";
 
-export interface GetProduct {
+export interface GetProducts {
   /** @description query to use on search */
   term?: string;
 
   /** @description Search by collection */
   collection?: number;
 
+  /** @description Search by category */
+  category?: number;
+
+  /** @description Search by subcategory */
+  subcategory?: number;
+
   /** @description total number of items to display*/
 
   count: number;
 
+  /** @description Dinamic filters Example: Aro Fechado,Retangular*/
+
+  dinamicFilters?: dinamicFilter[];
+
   /** @description Sort for product whit promotion */
   isStopwatch?: boolean;
 
   /** @description search sort parameter */
   sort: "none" | "nome";
 }
 
+export interface dinamicFilter {
```

Comment:
Dois pontos aqui: PascalCase para nomeaçao de tipos e um erro de digitaçao
```suggestion
export interface DynamicFilter {
```